pkgs/tools/networking/network-manager/libnma/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv
+, fetchurl
+, meson
+, ninja
+, gettext
+, gtk-doc
+, pkg-config
+, vala
+, networkmanager
+, gnome3
+, isocodes
+, libxml2
+, docbook_xsl
+, docbook_xml_dtd_43
+, mobile-broadband-provider-info
+, gobject-introspection
+, gtk3
+, withGnome ? true
+, gcr
+, glib
+, substituteAll
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libnma";
+  version = "1.8.28";
+
+  src = fetchurl {
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "09mp6k0hfam1vyyv9kcd8j4gb2r58i05ipx2nswb58ris599bxja";
+  };
+
+  mesonFlags = [
```

Comment:
Though maybe we could follow the traditional order as per https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887:

```
pname
version

outputs

src

patches

nativeBuildInputs

buildInputs

mesonFlags

postPatch

postInstall

passthru

meta
```

Reply:
Right, wasn't aware of a semi standard for ordering (apart from generally following the phase flow) :+1: